include/deal.II/numerics/kdtree.h

```diff
@@ -89,7 +89,7 @@ class KDTree
    * your points, and do not call set_points() again, then all following results
    * will likely be wrong.
    */
-  KDTree(const unsigned int &           max_leaf_size = 10,
+  KDTree(const unsigned int             max_leaf_size = 10,
          const std::vector<Point<dim>> &pts = std::vector<Point<dim>>());
```

Comment:
What about `const std::vector<Point<dim>> &pts = {}` while you are here? I am fine with either but we could avoid repeating the longish type and gain a nicer formatting.

Reply:
Can I leave this to someone else and another time? I don't have kdtree installed on the laptop I use here at the hackathon.

Reply:
Sure.